problem_builder/data_export.py

```diff
@@ -131,13 +156,34 @@ def _delete_export(self):
         self.active_export_task_id = ''
 
     @XBlock.json_handler
-    def start_export(self, request, suffix=''):
+    def start_export(self, data, suffix=''):
         """ Start a new asynchronous export """
+        block_types = data.get('block_types', None)
+        username = data.get('username', None)
+        root_block_id = data.get('root_block_id', None)
+        if not root_block_id:
+            root_block_id = unicode(self.scope_ids.usage_id)
+            get_root = True
+        else:
+            from xmodule.modulestore.django import modulestore
+            block = modulestore().get_items(self.runtime.course_id, qualifiers={'name': root_block_id}, depth=0)[0]
```

Comment:
@bradenmacdonald I considered that, but I ran into a little snag with the environment differences between the SDK and the LMS (which prompted some concerns about guarantees in the runtime). In this section, I want to get the ID of the intended root block, but not the full usage key, so that the code down the line could be the same. But I'd have to extract that ID from the usage ID there. In the workbench, the ID is unicode. In the LMS, the ID is a Usage Key. So in the LMS, I can get the block ID from that usage key, but in the workbench, that concept doesn't work.